geomstats/geometry/general_linear.py

```diff
@@ -30,6 +35,33 @@ def inv(point):
         """Return the inverse of a matrix."""
         return gs.linalg.inv(point)
 
+    def random_uniform(self, n_samples=1, tol=1e-8):
+        """Sample in GL(n) from the uniform distribution.
+
+        Parameters
+        ----------
+        n_samples: int, optional
```

Comment:
super-NIT: "n_samples : "
-> weirdly, there needs to be a space before the : in numpy's conventions for docstrings, which we are following.